.github/workflows/i18n-crowdin-download.yml

```diff
@@ -0,0 +1,51 @@
+name: Crowdin Download Action
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 * * * *"
```

Comment:
i know this schedule is mentioned in the issue, but once an hour feels a little aggressive. i would vote for relaxing it to more like once a week

Reply:
😅  The old schedule was once a day - this is just matching that.

Maybe we meet in the middle and say once a day? Once every 12 hours? 

My concern with once a week is that the infrequency could hide problems.

Reply:
yeah i don't super mind, so i'm happy to keep it like this initially then we can always tweak it later 👍

Reply:
I would go for once a day like the fast channel cadency, I think it fits but I do not have a lot of expertise to know if this is the better option for sure.

Reply:
I'll set a reminder for me to review the cadence in a couple of weeks.